.github/workflows/pr.yml

```diff
@@ -0,0 +1,37 @@
+name: Checkout and Test
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout nakama-dotnet
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+        repository: heroiclabs/nakama-dotnet
+
+    - name: Checkout nakama-client-testrunner
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+        repository: heroiclabs/nakama-client-testrunner
+
+    - name: Start docker containers for nakama-client-testrunner
+      working-directory: nakama-client-testrunner
+      run: ./docker-compose up -d
+
+    - name: Wait for the docker containers to start
+      working-directory: nakama-client-testrunner
+      run: |
```

Comment:
Open to any better suggestions on how to "wait" for the container to start.

Reply:
You can use the `--wait` flag command in the previous step to wait for the containers to become "healthy"

```
docker compose up -d --wait
```

Reply:
nice thanks